pages.hi/common/cksum.md

```diff
@@ -0,0 +1,9 @@
+# cksum
+
+> किसी फ़ाइल की सीआरसी चेकसम और बाइट गिनती की गणना करें।
+> ध्यान दें, पुराने यूनिक्स सिस्टम पर सीआरसी कार्यान्वयन भिन्न हो सकता है।
```

Comment:
Thanks for your contribution, I have a minor suggestion for this page.

```suggestion
> किसी फ़ाइल की CRC चेकसम और बाइट गिनती की गणना करें।
> ध्यान दें, पुराने UNIX सिस्टम पर CRC कार्यान्वयन भिन्न हो सकता है।
```
I wouldn't suggest transliterating abbreviations as oftentimes it may become confusing.